### ISSUE TITLE
Download a folder

### ISSUE THREAD

**Issue by meaz** [redmatrix/hubzilla#1156]
It would be nice if we could have the possibility to download a folder in the files app or the image app, and not just download image per image or file per file.

**Comment by zotlabs** [comment]
That's probably correct if you use the web app. 

Most of us use WebDAV where you can drag/drop files/folders either way from your operating system.

**Comment by meaz** [comment]
I agree, however, I think it would be easier for beginner users to have this download option.
